litgpt/finetune/adapter_v2.py

```diff
@@ -75,7 +75,8 @@ def setup(
         logger_name: The name of the logger to send metrics to.
         seed: The random seed to use for reproducibility.
     """
-
+    if not checkpoint_dir.is_dir() and not checkpoint_dir.parts[0] == "checkpoints":
+        checkpoint_dir = "checkpoints" / checkpoint_dir
```

Comment:
Thanks, I will update those for consistency. 

> Also, I think a and not checkpoint_dir.is_absolute() is missing, otherwise you'd prepend the "checkpoints" prefix to an absolute path if it is one.

Agreed. Thanks!